src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -110,8 +111,8 @@ class ActivitiesService(
     UpdateActivityRequest(
       endDate = endDate,
       internalLocationId = internalLocation?.id?.toLong(),
-      payRates = mapRates(pay),
-      scheduleRules = mapRules(this.slots),
+      payRates = pay.toPayRateRequests(),
+      scheduleRules = slots.toScheduleRuleRequests(),
```

Comment:
Small refactor to make all of the mapping methods the same format for consistency

Reply:
also more readable